src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -25,170 +25,185 @@ const propTypes = {
+            errors.dob = 'bankAccount.error.age';
         }
+    }
 
-        if (!ValidationUtils.isRequiredFulfilled(values.isControllingOfficer)) {
-            errors.isControllingOfficer = 'requestorStep.isControllingOfficerError';
-        }
+    if (values.ssnLast4 && !ValidationUtils.isValidSSNLastFour(values.ssnLast4)) {
+        errors.ssnLast4 = 'bankAccount.error.ssnLast4';
+    }
 
-        return errors;
+    if (values.requestorAddressStreet && !ValidationUtils.isValidAddress(values.requestorAddressStreet)) {
+        errors.requestorAddressStreet = 'bankAccount.error.addressStreet';
     }
 
-    submit(values) {
-        const payload = {
-            bankAccountID: lodashGet(this.props.reimbursementAccount, 'achData.bankAccountID') || 0,
-            ...values,
-        };
+    if (values.requestorAddressZipCode && !ValidationUtils.isValidZipCode(values.requestorAddressZipCode)) {
+        errors.requestorAddressZipCode = 'bankAccount.error.zipCode';
+    }
 
-        BankAccounts.updatePersonalInformationForBankAccount(payload);
+    if (!ValidationUtils.isRequiredFulfilled(values.isControllingOfficer)) {
+        errors.isControllingOfficer = 'requestorStep.isControllingOfficerError';
     }
 
-    render() {
-        if (this.props.shouldShowOnfido) {
-            return (
-                <RequestorOnfidoStep
-                    reimbursementAccount={this.props.reimbursementAccount}
-                    reimbursementAccountDraft={this.props.reimbursementAccountDraft}
-                    onBackButtonPress={this.props.onBackButtonPress}
-                    getDefaultStateForField={this.props.getDefaultStateForField}
-                />
-            );
-        }
+    return errors;
+};
 
+function InnerRequestorStep({reimbursementAccount, shouldShowOnfido, reimbursementAccountDraft, onBackButtonPress, getDefaultStateForField, translate}, ref) {
+    const defaultValues = useMemo(
+        () => ({
+            firstName: getDefaultStateForField('firstName'),
+            lastName: getDefaultStateForField('lastName'),
+            street: getDefaultStateForField('requestorAddressStreet'),
+            city: getDefaultStateForField('requestorAddressCity'),
+            state: getDefaultStateForField('requestorAddressState'),
+            zipCode: getDefaultStateForField('requestorAddressZipCode'),
+            dob: getDefaultStateForField('dob'),
+            ssnLast4: getDefaultStateForField('ssnLast4'),
+        }),
+        [getDefaultStateForField],
+    );
+
+    const submit = useCallback(
+        (values) => {
+            const payload = {
+                bankAccountID: lodashGet(reimbursementAccount, 'achData.bankAccountID') || 0,
+                ...values,
+            };
+
+            BankAccounts.updatePersonalInformationForBankAccount(payload);
+        },
+        [reimbursementAccount],
+    );
+
+    const LabelComponent = useCallback(
```

Comment:
```suggestion
    const renderLabelComponent = useCallback(
```

Reply:
Isn't a complex component or anything, I would not wrap it into a `useCallback`

Reply:
I've wrapped in `useCallback` with an idea to keep its reference stable for usage inside other component, in this case `CheckboxWithLabel`. I can drop it tho :)

Reply:
`CheckboxWithLabel` doesn't use it as a hook, I think we can drop it

Reply:
Done ✅